src/Controller/HealthController.php

```diff
@@ -0,0 +1,143 @@
+ * Authors: Conduction <info@conduction.nl> Barry Brands <barry@conduction.nl>
+ *
+ * @license EUPL <https://github.com/ConductionNL/contactcatalogus/blob/master/LICENSE.md>
+ *
+ * @category Controller
+ */
+class HealthController extends AbstractController
+{
+
+    /**
+     * @var MetricsService The metrics service
+     */
+    private MetricsService $metricsService;
+
+    /**
+     * @var ParameterBagInterface The parameterbaginterface
+     */
+    private ParameterBagInterface $parameterBagInterface;
+
+    /**
+     * @var EntityManagerInterface The entityManagerInterface
+     */
+    private EntityManagerInterface $entityManager;
+
+    /**
+     * The constructor sets al needed variables.
+     *
+     * @param MetricsService         $metricsService The metrics service
+     * @param ParameterBagInterface  $parameters     The Parameter bag
+     * @param EntityManagerInterface $entityManager  The Parameter bag
+     */
+    public function __construct(MetricsService $metricsService, ParameterBagInterface $parameters, EntityManagerInterface $entityManager)
+    {
+        $this->metricsService = $metricsService;
+        $this->parameters     = $parameters;
+        $this->entityManager  = $entityManager;
+
+    }//end __construct()
+
+    /**
+     * Provides a health check endpoint.
+     *
+     * @return Response
+     *
+     * @Route("/health-check", methods={"GET"})
+     */
+    public function healthCheck(Request $request): Response
+    {
+        $acceptHeader = ($request->headers->get('Accept') ?? 'application/json');
+
+        $status             = 'pass';
+        $outputMessage      = null;
+        $responseStatusCode = 200;
+        $mongoDBCacheStatus = 'pass';
+
+        try {
+            $client = new Client($this->parameters->get('cache_url'));
+            $client->listDatabases();
+        } catch (Exception | MongoDBException $e) {
+            $mongoDBCacheStatus = 'fail';
+            $status             = 'fail';
+            $outputMessage      = 'The cache MongoDB connection failed: '.$e->getMessage();
+            $responseStatusCode = 400;
+        }
+
+        $objectDatabases  = $this->entityManager->getRepository(Database::class)->findAll();
+        $mongoDBCount     = count($objectDatabases);
+        $mongoDBPassCount = 0;
+        $mongoDBFailCount = 0;
+
+        foreach ($objectDatabases as $database) {
+            try {
+                $client = new Client($database->getUri());
+                $client->listDatabases();
+                $mongoDBPassCount++;
+            } catch (Exception | MongoDBException $e) {
+                $mongoDBFailCount++;
+            }
+        }
+
+        if ($mongoDBFailCount > 0) {
+            if ($status !== 'fail') {
+                $status = 'warn';
+            }
+
+            $responseStatusCode = 400;
+            $message            = 'ome database connections failed: '.(string) $mongoDBFailCount.' out of '.(string) $mongoDBCount.' connections';
```

Comment:
Ome?

Reply:
In case he combines 2 messages he uses lower s and if this is the start of the message he uses capital S